### PR TITLE
[update/add][l] Adding new data fixing the codebase

### PR DESCRIPTION
**Pull request by gradedSystem**
## Changes made

- Updated `data/` and `cache/` folder
- Updated `README.md` and `datapackage.json`

cc @anuveyatsu

### CODE REVIEW

File: data/monthly.csv [hunk 1]
```diff
@@ -1,826 +1,2302 @@
...
-2018-01,1332.809
-2018-02,1333.775
-2018-03,1325.562
-2018-04,1335.332
-2018-05,1303.618
-2018-06,1282.126
-2018-07,1238.064
-2018-08,1201.859
-2018-09,1199.198
+Date,Gold
```
Comment (anuveyatsu) [inline data/monthly.csv]: @gradedSystem second column name should be `Price`

File: data/annual.csv [hunk 2]
```diff
@@ -1,69 +1,192 @@
...
-2009-12,1135.012
-2010-12,1393.512
-2011-12,1652.725
-2012-12,1687.342
-2013-12,1221.588
-2014-12,1200.440
-2015-12,1068.317
-2016-12,1152.165
-2017-12,1265.674
+Date,Gold
```
Comment (anuveyatsu) [inline data/annual.csv]: same here @gradedSystem re second column name - `Price`

Comment (gradedSystem) [reply data/annual.csv]: @anuveyatsu Updated

File: data/monthly.csv (hunk 1 above)
Comment (gradedSystem) [reply data/monthly.csv]: Updated

**Pull request merged**